src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/bootstrap-import-modal/bootstrap-import-modal.component.html

```diff
@@ -97,7 +97,9 @@
           <span *ngIf="importBootstrapForm.showError('token', formDir, 'invalidToken')"
                 class="invalid-feedback"
                 i18n>The token is invalid.</span>
+               
         </ng-template>
+          <span *ngIf="generateBtnDisable" class="invalid-feedback" i18n>Note : Pools are disabled either they already have peers or they have mirroring disabled </span>
```

Comment:
same here we need to put one attribute per line, sorry but its linter and we wont be able to merge this PR.
Thanks